src/com/xilinx/rapidwright/rwroute/RWRoute.java

```diff
@@ -289,14 +299,79 @@ private void determineRoutingTargets(){
 				System.err.println("ERROR: Unknown net " + net.toString());
 			}
 		}
-		if(this.config.isVerbose()) this.printConnectionSpanStatistics();
+	}
+	
+	/**
+	 * Deals with the conflicted nets.
+	 * Note: this is customized for the RapidStream use case. Other users will need to adapt this method.
+	 * @param design The design to route.
+	 * @param conflictNets A set of conflicted nets.
+	 */
+	private void handleConflictNets(Design design, Set<Net> conflictNets) {
+		List<Net> toPreserveNets = new ArrayList<>();
+		for(Net net : conflictNets) {
+			if(net.getType() != NetType.WIRE) {
+				// Skip successfully routed CLK, VCC, and GND nets
+				toPreserveNets.add(net);
+				continue;
+			}
+			
+			if(net.getSinkPins().size() > 1) {
+				// In the RapidStream flow, the target nets to route are 2-terminal FF-to-FF nets.
+				toPreserveNets.add(net);
+				continue;
+			}
+			
+			if(!this.isRegularAnchorNet(net, design)) {
+				toPreserveNets.add(net);
+				continue;
+			}
+			
+			this.removeNetNodesFromPreservedNodes(net); // remove preserved nodes of a net from the map
+			this.createsNetWrapperAndConnections(net, this.config.getBoundingBoxExtensionX(), this.config.getBoundingBoxExtensionY(), this.isMultiSLRDevice());
+			net.unroute();//NOTE: no need to unroute if routing tree is reused, then toPreserveNets should be detected before createNetWrapperAndConnections
+		}
+		for(Net net : toPreserveNets) {
+			this.preserveNet(net);
+		}
+	}
+	
+	/**
+	 * Checks if a net is an anchor net that is from / to a CLB tile.
+	 * Note: this is customized for the RapidStream use case.
+	 * @param net The net in question.
+	 * @param design The design to route.
+	 * @return true if the net is an anchor net from / to a CLB tile.
+	 */
+	private boolean isRegularAnchorNet(Net net, Design design) {
```

Comment:
Again, is the `design` parameter necessary here?

Reply:
Also, rather than simply using the term "Anchor" net, perhaps we could define the kind of situation that it is detecting so that others could find it useful.  If we state that it is only used for RapidStream, it will not be as easily leveraged.  Can you define in the Javadoc what kind of net it is detecting?